index.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var _ = require('lodash');
+var path = require('path');
+var fs = require('fs');
+var through2 = require('through2');
+var gettextParser = require('gettext-parser');
+var Handlebars = require('handlebars');
+
+
+_.templateSettings.interpolate = /{{trans\s"([\s\S]+?)"}}/g;
+
+var replaceText = function(catalog, chunk, enc, callback) {
+  var template = _.template(chunk.toString());
+
+  var translatedString = template(catalog)
+    .replace(/\n/g, '\\n')
+    .replace(/"/g, '\"');
+
```

Comment:
```js
.replace(/(\n|")/g, '\\$&');
```
this may also need to include some other things besides just new line chars and double quotes